github/list_private_repos.py

```diff
@@ -0,0 +1,53 @@
+import github3
+import csv
+import getpass
+
+code = ''
+try:
+    # Python 2
+    prompt = raw_input
+except NameError:
+    # Python 3
+    prompt = input
+
+def my_two_factor_function():
+    global code
+    if code == '':
+      code = prompt('Enter 2FA code: ')
+    return code
+  
+usr = prompt('Enter Username (has to be admin): ')
+pw = getpass.getpass()
+gh = github3.login(usr, pw,
+                  two_factor_callback=my_two_factor_function)
+org = gh.organization('ethz-asl')
+
+repos = org.iter_repos(type="private")
+
+with open('repos.csv', 'wb') as csvfile:
+  writer = csv.writer(csvfile, delimiter=',')
+  writer.writerow(["Name", "Created at", "Pushed at", "Updated at", "#1 contributor", "#2 contributor", "#3 contributor", "#4 contributor"])
```

Comment:
Python tip: `..] + [ "#%d contributor" % (i + 1) for i in range(numContributors) ]`